ssc/cmod_pvsamv1.cpp

```diff
@@ -772,6 +794,7 @@ static var_info _cm_vtab_pvsamv1[] = {
         //SEV: total dc snow loss time series (not a required output)
         { SSC_OUTPUT,        SSC_ARRAY,      "dc_snow_loss",                         "DC power loss due to snow",						 "kW",   "",   "Time Series (Array)",       "",                    "",                              "" },
         { SSC_OUTPUT,        SSC_ARRAY,      "dc_net",                               "Inverter DC input power",                                       "kW",   "",   "Time Series (Array)",       "*",                    "",                              "" },
+        //{ SSC_OUTPUT,        SSC_ARRAY,      "dc_net_clearsky",                               "Inverter DC clearsky input power",                                       "kW",   "",   "Time Series (Array)",       "",                    "",                              "" },
```

Comment:
Can we remove debug output if no longer needed?

Reply:
Removed.